html-forms/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <div>Email</div>
+      <input type="email" name="email" />
+
+      <div>Why is coding cool?</div>
+      <select name="cool-codes">
+        <option value="">Select an option</option>
+        <option value="Because it's interesting">
+          Because it's interesting
+        </option>
+        <option vlaue="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+
+      <div>
+        Favorite Language (choose one)
+        <br />
+        <input type="radio" name="codeTypes" value="html" /> HTML
+        <input type="radio" name="codeTypes" value="CSS" /> CSS
+        <input type="radio" name="codeTypes" value="JAVASCRIPT" /> JAVASCRIPT
```

Comment:
This text labeling the `input` should be in a `label` element.

Reply:
this has been updated similar to lines 30-31